recipes/aptardi/meta.yaml

```
@@ -0,0 +1,33 @@
+package:
+  name: aptardi
+  version: '1.4'
+
+source:
+  url: https://github.com/luskry/aptardi/archive/refs/tags/v.1.4.tar.gz
+  sha256: e7cae7e940573ffdb7db90cb856fdd38824e077ce0df88e95f3bf6a03ba61d43
```

It's good practice to use jinja variable wherever possible, so that you only have to update things in one place:

```suggestion
{% set name = "aptardi" %}
{% set version = "1.4" %}
{% set sha256 = "e7cae7e940573ffdb7db90cb856fdd38824e077ce0df88e95f3bf6a03ba61d43" %}

package:
  name: {{ name }}
  version: {{ version }}

source:
  url: https://github.com/luskry/{{ name }}/archive/refs/tags/v.{{ version }}.tar.gz
  sha256: {{ sha256 }}
```

Also, I think this makes it easier for the BiocondaBot to auto-update the recipe as soon as a new release appears on the repo.